src/main/groovy/life/qbic/datamodel/dtos/imaging/Instrument.groovy

```diff
@@ -1,12 +1,72 @@
 package life.qbic.datamodel.dtos.imaging
 
 /**
- * <class short description - 1 Line!>
+ * DTO that describes an instrument in the imaging context
  *
- * <More detailed description - When to use, what it solves, etc.>
+ * This class shall be used to describe some basic
+ * properties of an instrument that has been used to create an image
+ * during for example a microscopy or tomography analysis.
  *
  * @author Sven Fillinger
- * @since <versiontag>
+ * @since 1.10.0
  */
 class Instrument {
+
+  /**
+   * The manufacturer of the instrument.
+   * Examples are 'Leica', 'Zeiss' and many more.
+   */
+  private final String manufacturer
```

Comment:
do we want to have a controlled vocabulary here?

Reply:
The enumeration is unknown to me atm, so I used String